rest/api.go

```diff
@@ -182,39 +182,56 @@ func (h *handler) handlePostUpgrade() error {
 	return nil
 }
 
-func (h *handler) instanceStartTime() json.Number {
-	return json.Number(strconv.FormatInt(h.db.StartTime.UnixNano()/1000, 10))
+func instanceStartTime(dbc *db.DatabaseContext) json.Number {
```

Comment:
Refactored this to from a method to a function as if we're using this from `/_status` we can't rely on `h.db` and need to pass in the `DatabaseContext` explicitly.